### PR TITLE
Improve project search UI

### PR DESCRIPTION
**Pull request by johnnyd-eth**
## What does this PR do and why?

- Moves the V2 project notice below the search results
- Hides it when projects still loading
- Improves project count message padding-top when 0 projects

## Screenshots or screen recordings

Before:

https://user-images.githubusercontent.com/96150256/172064533-f12b14a4-62ab-4cac-ab9f-3289b28b01a2.mp4

After (excuse my terrible internet):

https://user-images.githubusercontent.com/96150256/172064549-6f27dde3-bfae-4b4b-bc13-433821eb0c5c.mp4

## Acceptance checklist

- [ ] I have evaluated the
      [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines)
      for this PR.
- [ ] I have tested this PR in
      [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).

### CODE REVIEW

File: src/components/Projects/index.tsx [hunk 1]
```diff
@@ -287,6 +274,18 @@ export default function Projects() {
           <TrendingProjects count={12} trendingWindowDays={7} />
         </div>
       ) : null}
+      {Boolean(searchText) && !isLoading && (
```
Comment (wraeth-eth) [inline src/components/Projects/index.tsx:277]: **question** is there a particular preference for this in the project? I am a bigger fan of the double bang (I think the airbnb standard also prefers it). I have seen both used in project

**Review by wraeth-eth — APPROVED** [review]
LGTM @johnnyd-eth ! Nice job, makes for a nicer UI. Only thing i'd raise is maybe having the v2 warning be shown while loading, but for most users the loading time is very short and won't impact them

File: src/components/Projects/index.tsx (hunk 1 above)
Comment (aeolianeth) [reply src/components/Projects/index.tsx:277]: Heh, I'm the opposite - Boolean() clear

Comment (aeolianeth) [reply src/components/Projects/index.tsx:277]: Ultimately: it's whatever the linter allows. Lets lint for this if we feel strongly

**Pull request merged**